js/books.js

```diff
@@ -0,0 +1,62 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const booklist = document.getElementsByClassName('book-list')[0];
+const form = document.getElementsByClassName('book-form')[0];
+
+let allBook = [];
+
+/**
+ *
+ */
+if (localStorage.getItem('ourbook') != null) {
+  allBook = JSON.parse(localStorage.getItem('ourbook'));
+}
+
+let index = 0;
+for (let i = 0; i < allBook.length; i += 1) {
+  const book = allBook[i];
+  const bookTemplate = `<div class="book">
+        <p class="book-title">${book.title}</p>
+        <p class="book-author">${book.author}</p>
+        <button type="button" id="${index}" class="btn-remove">
+    Remove
+  </button>
+<br/>
+<br/>
+  <hr>
+    </div>`;
+
+  const domBook = new DOMParser().parseFromString(bookTemplate, 'text/html');
+  const bookContainer = domBook.querySelector('.book');
+  booklist.appendChild(bookContainer);
+  index += 1;
+}
+
+const btnRemove = document.querySelectorAll('.book .btn-remove');
+
+function addBook(event) {
```

Comment:
While adding the book, I am getting this error. Consider fixing it and then submit it for a re-review.
![image](https://user-images.githubusercontent.com/33268927/182351724-39cdb815-8602-4ed1-91dd-21bf9925ff26.png)